classes/classes/Items/Mutations.as

```diff
@@ -1265,6 +1253,21 @@
 				player.skinDesc = "fur";
 				player.underBody.restore(); // Restore the underbody for now
 			}
+			// Hooves - Tail and not taur
+			if (player.lowerBody != LOWER_BODY_TYPE_HOOFED && !player.isTaur() && player.tailType == TAIL_TYPE_HORSE && changes < changeLimit && rand(5) == 0) {
```

Comment:
Care to remove the `!isTaur()` requirement here?

Since the overdose bad end now requires the hoofs being tauric and having a non-hoofed lowerBody can lock you out of the overdose bad end. You shouldn't be able the cheat yourself out of a bad end, right? ;-)